src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -573,13 +560,15 @@ private static StringBuilder fakeBody(final Request... parts)
                     ),
                     String.format(
                         "Content-Length: %s",
-                        RqMultipart.Fake.fakeBody(this.parts).length()
+                        this.body.length()
                     )
                 ).head();
             }
             @Override
             public InputStream body() throws IOException {
```

Comment:
@dalifreire this doesn't thrown `IOException` anymore now, you're just creating a stream on a String`s bytes :)